front/src/i18n/zh-CN/index.ts

```diff
@@ -0,0 +1,34 @@
+import type { BaseTranslation } from "../i18n-types";
+import audio from "./audio";
+import camera from "./camera";
+import chat from "./chat";
+import companion from "./companion";
+import woka from "./woka";
+import error from "./error";
+import follow from "./follow";
+import login from "./login";
+import menu from "./menu";
+import report from "./report";
+import warning from "./warning";
+import emoji from "./emoji";
+import trigger from "./trigger";
+
+const zh_CN: BaseTranslation = {
```

Comment:
@moufmouf I think this is not supposed to be a `BaseTranslation` as there should only be one base translation, right?